contracts/staking/ValidatorManager.sol

```diff
@@ -204,6 +204,11 @@ abstract contract ValidatorManager is
         );
     }
 
+    function activeValidators(bytes32 nodeID) public view returns (bytes32) {
```

Comment:
Can we rename this to `getActiveValidatorID` or similar? The current name implies we return a collection of validators.

Reply:
i went with `activeValidators` to simulate the default getter function you'd have with normal public state variables

Reply:
Yes, my only issue with that is that this getter doesn't return the `activeValidators` state variable, but instead returns a value from that mapping.

Reply:
Bumping this comment.

Reply:
the default getter for a public mapping state varaible would act the same, the getter function returns the value associated with a passed in key, not the whole state variable.

Reply:
Ah, I did not realize that. Personal preference for names aside, sticking to established Solidity convention is the right call.